packages/mds-policy-service/repository/index.ts

```diff
@@ -431,10 +432,14 @@ class PolicyReadWriteRepository extends ReadWriteRepository {
     }
   }
 
-  public updatePolicyMetadata = async <M>(metadata: PolicyMetadataDomainModel<M>) => {
+  public updatePolicyMetadata = async <M>(metadata: PolicyMetadataDomainUpdateModel<M>) => {
     try {
-      const { policy_id, policy_metadata } = metadata
-      await this.readSinglePolicyMetadata(policy_id)
+      const { policy_id, template, policy_metadata } = metadata
+      const policyMetadata = await this.readSinglePolicyMetadata(policy_id)
```

Comment:
nit - maybe call this one something like `oldPolicyMetadata` to avoid confusion?

Reply:
Code removed since no longer needed with AC change. See 72bbf98